### PR TITLE
Unify the wording for each query pack

### PR DESCRIPTION
**Pull request by tas50**
We're all over the place with how we talk about the data we're gathering in cnquery. Most places we say we're "gathering" data so let's use that everywhere. We don't need to say we're gathering a list or gathering information since that's just assumed. Just stick to a consistent and basic form so the output on the CLI looks clean.

### CODE REVIEW

**Comment by misterpantz** [comment]
It's awesome to standardize wording! 🎉  "Gather" doesn't work for single values. "Gather" means "bring together." How do you feel about standardizing on "retrieve"?

**Comment by tas50** [comment]
Switched it all to retrieves. Thanks for the feedback @misterpantz

File: core/mondoo-aws-incident-response.mql.yaml [hunk 1]
```diff
@@ -88,7 +88,7 @@ packs:
         title: Retrieve data on IAM Users, Groups, and Roles where AdministratorAccess policy is attached
         docs:
           desc: |
-            This query retrieves all IAM Users, Groups, and Roles that have the `AdministratorAccess` role attached.
+            This query retrieves all IAM Users, Groups, and Roles with the `AdministratorAccess` role attached.
```
Comment (misterpantz) [inline core/mondoo-aws-incident-response.mql.yaml:91]:
```suggestion
            This query retrieves all IAM users, groups, and roles with the `AdministratorAccess` role attached.
```

File: core/mondoo-aws-inventory.mql.yaml [hunk 2]
```diff
@@ -55,7 +55,7 @@ packs:
         title: Retrieve data on all attached IAM Policies
         docs:
           desc: |
-            This query retrieves all IAM policies that are attached to either a User, Group, or Role.
+            This query retrieves all IAM policies attached to a User, Group, or Role.
```
Comment (misterpantz) [inline core/mondoo-aws-inventory.mql.yaml:58]:
```suggestion
            This query retrieves all IAM policies attached to a user, group, or role.
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ packs:
         ## Join the community!
         Our goal is to build query packs that are simple to deploy, and provide accurate and useful data. 
         
-        If you have any suggestions on how to improve this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
+        If you have any suggestions on improving this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions.
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:24]:
```suggestion
        If you have any suggestions for improving this query pack, or if you need support, [join the Mondoo community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
```

File: core/mondoo-linux-inventory.mql.yaml [hunk 4]
```diff
@@ -29,7 +29,7 @@ packs:
         ## Join the community!
         Our goal is to build query packs that are simple to deploy, and provide accurate and useful data. 
         
-        If you have any suggestions on how to improve this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
+        If you have any suggestions on improving this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions.
```
Comment (misterpantz) [inline core/mondoo-linux-inventory.mql.yaml:32]:
```suggestion
        If you have any suggestions for improving this query pack, or if you need support, [join the Mondoo community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
```

File: core/mondoo-macos-inventory.mql.yaml [hunk 5]
```diff
@@ -29,7 +29,7 @@ packs:
         ## Join the community!
         Our goal is to build query packs that are simple to deploy, and provide accurate and useful data. 
         
-        If you have any suggestions on how to improve this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
+        If you have any suggestions on improving this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions.
```
Comment (misterpantz) [inline core/mondoo-macos-inventory.mql.yaml:32]:
```suggestion
        If you have any suggestions for improving this query pack, or if you need support, [join the Mondoo community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
```

File: core/mondoo-vmware-incident-response.mql.yaml [hunk 6]
```diff
@@ -26,7 +26,7 @@ packs:
 
         Our goal is to build policies that are simple to deploy, accurate, and actionable. 
         
-        If you have any suggestions on how to improve this policy, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions.
+        If you have any suggestions on improving this policy, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions.
```
Comment (misterpantz) [inline core/mondoo-vmware-incident-response.mql.yaml:29]:
```suggestion
        If you have any suggestions for improving this policy, or if you need support, [join the Mondoo community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions.
```

File: core/mondoo-windows-inventory.mql.yaml [hunk 7]
```diff
@@ -29,7 +29,7 @@ packs:
         ## Join the community!
         Our goal is to build query packs that are simple to deploy, and provide accurate and useful data. 
         
-        If you have any suggestions on how to improve this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
+        If you have any suggestions on improving this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions.
```
Comment (misterpantz) [inline core/mondoo-windows-inventory.mql.yaml:32]:
```suggestion
        If you have any suggestions for improving this query pack, or if you need support, [join the Mondoo community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
```

**Review by misterpantz — APPROVED** [review]
Good changes, @tas50. I made a few more.

File: README.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
-# cnquery-packs
+# cnquery Query Packs
 
-This project contains query packs used with [`cnquery`](https://github.com/mondoohq/cnquery). We've organized them into these directories:
+This project contains inventory and incident response query packs for [`cnquery`](https://github.com/mondoohq/cnquery). We've organized them into these directories:
 
 - [core](core) - Core packs contain baseline queries such as incidence response probes for different targets. Core query packs are maintained by Mondoo and have strict quality requirements.
```
Comment (misterpantz) [inline README.md:5]:
```suggestion
- [core](core) - Core packs contain baseline queries such as incident response probes for different targets. Core query packs are maintained by Mondoo and have strict quality requirements.
```

File: core/mondoo-aws-incident-response.mql.yaml [hunk 9]
```diff
@@ -97,10 +97,10 @@ packs:
               attachedRoles 
             }
       - uid: mondoo-incident-response-aws-iam-full-access
-        title: Gather data on IAM Users, Groups, and Roles where any 'FullAccess' policy is attached
+        title: Retrieve data on IAM Users, Groups, and Roles where any 'FullAccess' policy is attached
```
Comment (misterpantz) [inline core/mondoo-aws-incident-response.mql.yaml:100]:
```suggestion
        title: Retrieve data on IAM users, groups, and roles to which any 'FullAccess' policy is attached
```

File: core/mondoo-aws-incident-response.mql.yaml [hunk 10]
```diff
@@ -97,10 +97,10 @@ packs:
...
         docs:
           desc: |
-            This query gathers all IAM Users, Groups, and Roles that have any one of the AWS FullAccess roles attached.
+            This query retrieves all IAM Users, Groups, and Roles that have any one of the AWS FullAccess roles attached.
```
Comment (misterpantz) [inline core/mondoo-aws-incident-response.mql.yaml:103]:
```suggestion
            This query retrieves all IAM users, croups, and roles that have any one of the AWS FullAccess roles attached.
```

File: core/mondoo-aws-incident-response.mql.yaml [hunk 11]
```diff
@@ -85,10 +85,10 @@ packs:
               accessKey2LastRotated
             }
       - uid: mondoo-incident-response-aws-iam-administrator-access
-        title: Gather data on IAM Users, Groups, and Roles where AdministratorAccess policy is attached
+        title: Retrieve data on IAM Users, Groups, and Roles where AdministratorAccess policy is attached
```
Comment (misterpantz) [inline core/mondoo-aws-incident-response.mql.yaml:88]:
```suggestion
        title: Retrieve data on IAM users, groups, and roles to which the AdministratorAccess policy is attached
```

File: core/mondoo-gcp-inventory.mql.yaml [hunk 12]
```diff
@@ -25,70 +25,70 @@ packs:
...
-        title: Gather data from all running GCP compute instances in a GCP project
+        title: Retrieve data from all running GCP compute instances in a GCP project
         docs: 
           desc: |
-            This query gathers the data for all running GCP compute instances in a GCP project
+            This query retrieves the data for all running GCP compute instances in a GCP project
         query: gcp.compute.instances.where( status == "RUNNING" ) { * }
       - uid: mondoo-asset-inventory-gcp-compute-instances-public
-        title: Gather data on public GCP Compute instances
+        title: Retrieve data on public GCP Compute instances
```
Comment (misterpantz) [inline core/mondoo-gcp-inventory.mql.yaml:76]:
```suggestion
        title: Retrieve data on public GCP Compute Engine instances
```

File: core/mondoo-gcp-inventory.mql.yaml [hunk 13]
```diff
@@ -25,70 +25,70 @@ packs:
...
         docs:
           desc: |
-            This query gathers the data for all GCP compute instances that have been configured with an external IP address. 
+            This query retrieves the data for all GCP compute instances that have been configured with an external IP address.
```
Comment (misterpantz) [inline core/mondoo-gcp-inventory.mql.yaml:79]:
```suggestion
            This query retrieves the data for all GCP Compute Engine instances that have been configured with an external IP address. 
```

File: core/mondoo-gcp-inventory.mql.yaml [hunk 14]
```diff
@@ -25,70 +25,70 @@ packs:
...
         docs:
           desc: |
-            This query gathers the data for all GCP compute instances that have been configured with an external IP address. 
+            This query retrieves the data for all GCP compute instances that have been configured with an external IP address. 
         query: |
           gcp.compute.instances
             .where( networkInterfaces[0]['accessConfigs'][0]['name'] == "External NAT" ) { * }
       - uid: mondoo-asset-inventory-gcp-compute-networks-count
-        title: Gather a count of GCP compute networks in a GCP project
+        title: Retrieve a count of GCP compute networks in a GCP project
```
Comment (misterpantz) [inline core/mondoo-gcp-inventory.mql.yaml:84]:
```suggestion
        title: Retrieve a count of GCP Compute Engine networks in a GCP project
```

File: core/mondoo-gcp-inventory.mql.yaml [hunk 15]
```diff
@@ -25,70 +25,70 @@ packs:
...
         docs: 
           desc: |
-            This query gathers a count of GCP compute networks configured in a GCP project
+            This query retrieves a count of GCP compute networks configured in a GCP project
```
Comment (misterpantz) [inline core/mondoo-gcp-inventory.mql.yaml:87]:
```suggestion
            This query retrieves a count of GCP Compute Engine networks configured in a GCP project
```

File: core/mondoo-gcp-inventory.mql.yaml [hunk 16]
```diff
@@ -25,70 +25,70 @@ packs:
...
         query: gcp.compute.networks.length
       - uid: mondoo-asset-inventory-gcp-compute-networks-data
-        title: Gather data for all GCP compute networks in a GCP project
+        title: Retrieve data for all GCP compute networks in a GCP project
```
Comment (misterpantz) [inline core/mondoo-gcp-inventory.mql.yaml:90]:
```suggestion
        title: Retrieve data for all GCP Compute Engine networks in a GCP project
```

File: core/mondoo-gcp-inventory.mql.yaml [hunk 17]
```diff
@@ -25,70 +25,70 @@ packs:
...
         docs: 
           desc: |
-            This query gathers the data for all GCP compute networks configured in a GCP project
+            This query retrieves the data for all GCP compute networks configured in a GCP project
```
Comment (misterpantz) [inline core/mondoo-gcp-inventory.mql.yaml:93]:
```suggestion
            This query retrieves the data for all GCP Compute Engine networks configured in a GCP project.
```

File: core/mondoo-github-incident-response.mql.yaml [hunk 18]
```diff
@@ -12,7 +12,7 @@ packs:
       desc: |
         ### Overview
 
-        The GitHub Organization Incident Response Pack by Mondoo query pack gathers configuration data about GitHub Organizations and the repositories within for investigation during a security incident.
+        The GitHub Organization Incident Response Pack by Mondoo query pack retrieves configuration data about GitHub Organizations and the repositories within for investigation during a security incident.
```
Comment (misterpantz) [inline core/mondoo-github-incident-response.mql.yaml:15]:
```suggestion
        The GitHub Organization Incident Response Pack by Mondoo query pack retrieves configuration data about GitHub organizations and the repositories within them for investigation during a security incident.
```

File: core/mondoo-github-incident-response.mql.yaml [hunk 19]
```diff
@@ -95,7 +95,7 @@ packs:
         title: Retrieve GitHub Organization private repositories
         docs: 
           desc: |
-            This query retrieves all of the repositories within the GitHub organization that are public. The query returns the name of the repo, and returns whether the default branch is [protected](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches) using protection rules. 
+            This query retrieves all of the public repositories within the GitHub organization. The query returns the repo's name and returns whether the default branch is [protected](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches) using protection rules.
```
Comment (misterpantz) [inline core/mondoo-github-incident-response.mql.yaml:98]:
```suggestion
            This query retrieves all of the public repositories within the GitHub organization. The query returns the repo's name and whether the default branch is [protected](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches) using protection rules.
```

File: core/mondoo-github-inventory.mql.yaml [hunk 20]
```diff
@@ -12,7 +12,7 @@ packs:
       desc: |
         ### Overview
 
-        The GitHub Organization Inventory Pack by Mondoo query pack gathers configuration data about GitHub organization.
+        The GitHub Organization Inventory Pack by Mondoo query pack retrieves configuration data about GitHub organization.
```
Comment (misterpantz) [inline core/mondoo-github-inventory.mql.yaml:15]:
```suggestion
        The GitHub Organization Inventory Pack by Mondoo query pack retrieves configuration data about GitHub organizations.
```

File: core/mondoo-kubernetes-incident-response.mql.yaml [hunk 21]
```diff
@@ -12,11 +12,11 @@ packs:
       - asset.platform == "kubernetes" || asset.platform == "k8s-cluster"
     queries:
       - uid: k8s-cluster-version
-        title: Gather Kubernetes Cluster Version
+        title: Retrieve Kubernetes Cluster Version
         query: |
           k8s.serverVersion
       - uid: role-bindings-with-cluster-admin-permissions
-        title: Gather Role Bindings with cluster-admin Permissions
+        title: Retrieve Role Bindings with cluster-admin Permissions
```
Comment (misterpantz) [inline core/mondoo-kubernetes-incident-response.mql.yaml:19]:
```suggestion
        title: Retrieve role bindings with cluster-admin permissions
```

File: core/mondoo-kubernetes-incident-response.mql.yaml [hunk 22]
```diff
@@ -25,7 +25,7 @@ packs:
             roleRef
           }
       - uid: clusterrole-bindings-with-cluster-admin-permissions
-        title: Gather ClusterRole Bindings with cluster-admin Permissions
+        title: Retrieve ClusterRole Bindings with cluster-admin Permissions
```
Comment (misterpantz) [inline core/mondoo-kubernetes-incident-response.mql.yaml:28]:
```suggestion
        title: Retrieve ClusterRoleBindings with cluster-admin permissions
```

File: core/mondoo-kubernetes-incident-response.mql.yaml [hunk 23]
```diff
@@ -45,7 +45,7 @@ packs:
       - asset.platform == "k8s-pod"
     queries:
       - uid: k8s-pod-security-context
-        title: Gather Pods Security Context
+        title: Retrieve Pods Security Context
```
Comment (misterpantz) [inline core/mondoo-kubernetes-incident-response.mql.yaml:48]:
```suggestion
        title: Retrieve Pods' Security Context
```

File: core/mondoo-kubernetes-incident-response.mql.yaml [hunk 24]
```diff
@@ -59,7 +59,7 @@ packs:
             }
           }
       - uid: k8s-pod-container
-        title: Gather Container Image Information
+        title: Retrieve Container Image Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-incident-response.mql.yaml:62]:
```suggestion
        title: Retrieve container image information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 25]
```diff
@@ -21,24 +21,24 @@ packs:
...
         Our goal is to build query packs that are simple to deploy, and provide accurate and useful data. 
         
-        If you have any suggestions on how to improve this query pack, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
+        If you have any suggestions for improving this query pack, or if you need support, [join the Mondoo community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
     filters:
       - asset.platform == "kubernetes" || asset.platform == "k8s-cluster"
     queries:
       - uid: k8s-cluster-version
-        title: Gather Kubernetes Cluster Version
+        title: Retrieve Kubernetes Cluster Version
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:29]:
```suggestion
        title: Retrieve Kubernetes cluster version
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 26]
```diff
@@ -21,24 +21,24 @@ packs:
...
         query: |
           k8s.serverVersion
       - uid: k8s-cluster-namespaces
-        title: Gather Kubernetes Cluster Namespaces
+        title: Retrieve Kubernetes Cluster Namespaces
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:33]:
```suggestion
        title: Retrieve Kubernetes cluster namespaces
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 27]
```diff
@@ -21,24 +21,24 @@ packs:
...
         query: |
           k8s.namespaces{ * }
       - uid: k8s-cluster-nodes
-        title: Gather Cluster Nodes
+        title: Retrieve Cluster Nodes
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:37]:
```suggestion
        title: Retrieve cluster modes
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 28]
```diff
@@ -21,24 +21,24 @@ packs:
...
         query: |
           k8s.nodes { * }
       - uid: k8s-cluster-clusterroles
-        title: Gather Cluster RBAC ClusterRoles
+        title: Retrieve Cluster RBAC ClusterRoles
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:41]:
```suggestion
        title: Retrieve cluster RBAC ClusterRoles
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 29]
```diff
@@ -54,11 +54,11 @@ packs:
       - asset.platform == "k8s-pod"
     queries:
       - uid: k8s-pod
-        title: Gather Pod Information
+        title: Retrieve Pod Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:57]:
```suggestion
        title: Retrieve Pod information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 30]
```diff
@@ -54,11 +54,11 @@ packs:
...
         query: |
           k8s.pod { * }
       - uid: k8s-pod-container
-        title: Gather Container Information
+        title: Retrieve Container Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:61]:
```suggestion
        title: Retrieve container information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 31]
```diff
@@ -74,11 +74,11 @@ packs:
       - asset.platform == "k8s-deployment"
     queries:
       - uid: k8s-deployment
-        title: Gather Deployment Information
+        title: Retrieve Deployment Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:77]:
```suggestion
        title: Retrieve deployment information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 32]
```diff
@@ -74,11 +74,11 @@ packs:
...
         query: |
           k8s.deployment { * }
       - uid: k8s-deployment-container
-        title: Gather Container Information
+        title: Retrieve Container Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:81]:
```suggestion
        title: Retrieve container information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 33]
```diff
@@ -94,11 +94,11 @@ packs:
       - asset.platform == "k8s-cronjob"
     queries:
       - uid: k8s-cronjob
-        title: Gather CronJob Information
+        title: Retrieve CronJob Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:97]:
```suggestion
        title: Retrieve CronJob information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 34]
```diff
@@ -94,11 +94,11 @@ packs:
...
         query: |
           k8s.cronjob { * }
       - uid: k8s-cronjob-container
-        title: Gather Container Information
+        title: Retrieve Container Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:101]:
```suggestion
        title: Retrieve container information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 35]
```diff
@@ -114,11 +114,11 @@ packs:
       - asset.platform == "k8s-job"
     queries:
       - uid: k8s-job
-        title: Gather Job Information
+        title: Retrieve Job Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:117]:
```suggestion
        title: Retrieve Job information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 36]
```diff
@@ -114,11 +114,11 @@ packs:
...
         query: |
           k8s.job { * }
       - uid: k8s-job-container
-        title: Gather Container Information
+        title: Retrieve Container Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:121]:
```suggestion
        title: Retrieve container information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 37]
```diff
@@ -134,11 +134,11 @@ packs:
       - asset.platform == "k8s-daemonset"
     queries:
       - uid: k8s-daemonset
-        title: Gather DaemonSet Information
+        title: Retrieve DaemonSet Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:137]:
```suggestion
        title: Retrieve DaemonSet information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 38]
```diff
@@ -134,11 +134,11 @@ packs:
...
         query: |
           k8s.daemonset { * }
       - uid: k8s-daemonset-container
-        title: Gather Container Information
+        title: Retrieve Container Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:141]:
```suggestion
        title: Retrieve container information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 39]
```diff
@@ -154,11 +154,11 @@ packs:
       - asset.platform == "k8s-statefulset"
     queries:
       - uid: k8s-statefulset
-        title: Gather StatefulSet Information
+        title: Retrieve StatefulSet Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:157]:
```suggestion
        title: Retrieve StatefulSet information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 40]
```diff
@@ -154,11 +154,11 @@ packs:
...
         query: |
           k8s.statefulset { * }
       - uid: k8s-statefulset-container
-        title: Gather Container Information
+        title: Retrieve Container Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:161]:
```suggestion
        title: Retrieve Container information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 41]
```diff
@@ -174,12 +174,10 @@ packs:
       - asset.platform == "k8s-replicaset"
     queries:
       - uid: k8s-replicaset
-        title: Gather ReplicaSet Information
+        title: Retrieve ReplicaSet Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:177]:
```suggestion
        title: Retrieve ReplicaSet information
```

File: core/mondoo-kubernetes-inventory.mql.yaml [hunk 42]
```diff
@@ -174,12 +174,10 @@ packs:
...
         query: |
           k8s.replicaset { * }
       - uid: k8s-replicaset-container
-        title: Gather Container Information
+        title: Retrieve Container Information
```
Comment (misterpantz) [inline core/mondoo-kubernetes-inventory.mql.yaml:181]:
```suggestion
        title: Retrieve container information
```

File: core/mondoo-ssl-tls-certificate-incident-response.mql.yaml [hunk 43]
```diff
@@ -33,22 +33,22 @@ packs:
...
-        title: Retrieve information about signature algorithm of all certificates in the certificate chain
+        title: Retrieve the signature algorithm of all certificates in the certificate chain
         query: |
           tls.certificates {
             signingAlgorithm
             subject.commonName
           }
       - uid: mondoo-ssl-tls-certificate-incident-response-is-revoked
-        title: Retrieve information about revoked, verified and CA status of all certificates in the certificate chain
+        title: Retrieve revoked, verified and CA status of all certificates in the certificate chain
```
Comment (misterpantz) [inline core/mondoo-ssl-tls-certificate-incident-response.mql.yaml:51]:
```suggestion
        title: Retrieve revoked, verified, and CA status of all certificates in the certificate chain
```

File: extra/mondoo-okta-incident-response.mql.yaml [hunk 44]
```diff
@@ -12,7 +12,7 @@ packs:
       desc: |
         ### Overview
 
-        The Okta Incident Response query pack gathers configuration data about your Okta investigation during a security incident.
+        The Okta Incident Response query pack retrieves configuration data about your Okta investigation during a security incident.
```
Comment (misterpantz) [inline extra/mondoo-okta-incident-response.mql.yaml:15]:
```suggestion
        The Okta Incident Response query pack retrieves configuration data about your Okta organization during a security incident.
```

File: extra/mondoo-slack-incident-response.mql.yaml [hunk 45]
```diff
@@ -12,7 +12,7 @@ packs:
       desc: |
         ### Overview
 
-        The Slack Incident Response query pack gathers configuration data about your Slack investigation during a security incident.
+        The Slack Incident Response query pack retrieves configuration data about your Slack investigation during a security incident.
```
Comment (misterpantz) [inline extra/mondoo-slack-incident-response.mql.yaml:15]:
```suggestion
        The Slack Incident Response query pack retrieves configuration data about your Slack organization during a security incident.
```

File: extra/mondoo-googleworkplace-incident-response.mql.yaml [hunk 46]
```diff
@@ -12,7 +12,7 @@ packs:
       desc: |
         ### Overview
 
-        The Google Workspace Incident Response query pack gathers configuration data about your Google Workspace investigation during a security incident.
+        The Google Workspace Incident Response query pack retrieves configuration data about your Google Workspace investigation during a security incident.
```
Comment (misterpantz) [inline extra/mondoo-googleworkplace-incident-response.mql.yaml:15]:
```suggestion
        The Google Workspace Incident Response query pack retrieves configuration data about your Google Workspace configuration during a security incident.
```

**Pull request merged**